### ISSUE TITLE
Create Sandbox

### ISSUE THREAD

**Issue by ravi-prakash-v** [beckn/online-dispute-resolution#1]
1. Document Specs
2. Document Architecture
3. Build BAP Sandbox API
4. Build BPP Sandbox API
5. Test using reference applications
6. Deploy Sandbox

**Comment by Mohit-Aasirwal** [comment]
Hi!
I would love to explore this project more...

**Comment by abhibamnote** [comment]
Hi!
@faizmagic @hemant-eminds @nikhil2128 @ujjwal502 I would like to know more about this project.

**Comment by Flanker-shyam** [comment]
Greetings mentors,

This is Shyam Sunder, I found Beckn protocols and its working super exciting, the idea is big and intuitive, the idea of shifting online dispute resolution on the ONDC network is a very effective step by our government and It will be a great opportunity for me to contribute on such a growing tech.

As I am a Beckend developer and I work with javascript, Nodejs and mongoDB the tech stacks of this project suits me and I would love to work on this project.

Regards,
Shyam Sunder
[Github profile](https://github.com/Flanker-shyam)

**Comment by Flanker-shyam** [comment]
Here is my understanding for the same, kindly correct me If I am wrong,

The way other sandboxes working, like that of mobility, we need to create one for ODR also, In which we need to understand the use cases of the ODR field and then Implement them.

Like one of the common use case of ODR will be to find the ODR services and the portals, In that case I need to change the fields in search end point, like what will be the context in the scenario and the message field of the same, then In BAP side, we need to modify in similar way that what data we are going to send, first to the BG and then to the BAP..

like it is implemented here !

**Comment by Flanker-shyam** [comment]
> @Flanker-shyam Surely you must explain very well , but can you please explain bit more about , what is this project about , scope etc . because I am new to this , I didn't get it yet .

Yeah sure @HarshPanchal26 let's discuss it, you can DM me on discord here at Flanker^Shyam#6675

**Comment by dhirajshettyyy** [comment]
@abhibamnote @Mohit-Aasirwal @Flanker-shyam @HarshPanchal26 ,
It's great to see your interest in this project, you can make an active contribution via issue creation and by submitting your Pull Requests.

For support from fellow contributors, Join Beckn Open Collective server :https://bit.ly/bocWebInvite

**Comment by HarshPanchal26** [comment]
@Flanker-shyam  I text you on 

> > @Flanker-shyam Surely you must explain very well , but can you please explain bit more about , what is this project about , scope etc . because I am new to this , I didn't get it yet .
> 
> Yeah sure @HarshPanchal26 let's discuss it, you can DM me on discord here at Flanker^Shyam#6675

Yes @Flanker-shyam  I send DM you on discord , Can you check it ,and we can start working on it.

**Comment by ravi-prakash-v** [comment]
@emmayank please add the PULSE sandbox link with instructions on how to use it to the README.

**Comment by emmayank** [comment]
@ravi-prakash-v , done with the changes and raised the PR
